package.json

```diff
@@ -4,10 +4,12 @@
   "description": "",
   "main": "index.js",
   "scripts": {
+    "start": "npm-run-all --sequential build start:backend",
+    "start:backend": "node ./src/server/index.js",
+    "build": "webpack -d --colors --config webpack.prod.js",
+    "dev:frontend": "webpack-dev-server --colors --config webpack.dev.js --port 3000",
+    "dev:backend": "nodemon ./src/server/index.js",
     "test:backend": "mocha",
-    "test:frontend": "jest --colors",
```

Comment:
why did you remove our test command?

Reply:
I commit before you merge so I don't have the test command